app/src/main/java/com/example/tung/moviedb_07/screen/list_item/ListItemViewModel.java

```diff
@@ -1,8 +1,12 @@
 package com.example.tung.moviedb_07.screen.list_item;
 
+import com.example.tung.moviedb_07.data.model.Genre;
+import com.example.tung.moviedb_07.data.model.Movie;
```

Comment:
ERROR: Unused import: com.example.tung.moviedb_07.data.model.Movie. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports